src/components/Tutorial/TutorialScrollUpButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, useCallback } from 'react'
+import scrollTo from '../../device/scrollTo'
+import useScrollPosition from '../../hooks/useScrollPosition'
+import TutorialNavigationButton from './TutorialNavigationButton'
+
+/** A button to scroll up to the tutorial. */
+const TutorialScrollUpButton: FC<{ show: boolean }> = ({ show }) => {
+  const scrollPosition = useScrollPosition()
```

Comment:
If we stick with subscribing directly to the scroll event, we should only enable it when the button is visible for performance reasons.

If we switch to the scrollTopStore, we can leave it as is.

Reply:
I've switched to use the `scrollTopStore` 👍